src/utils/helpers/index.jsx

```diff
@@ -5,6 +5,11 @@ export const getDate = () => {
   return new Date().toLocaleDateString('en-GB').replace(/\//g, '.');
 };
 
+export const daysBetweenDate = (date) => {
+  const diffTime = Math.abs(new Date() - date);
+  return Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+};
+
 export const sanitize = (text) => {
   return DOMPurify.sanitize(text);
```

Comment:
avoid using a third party library.

Reply:
I think, It was suggested by the mentors. Could you please have a look: https://github.com/ahr-wtag/react-todo-app/pull/1#discussion_r1172213943

Reply:
I have removed `DOMPurify` library. thanks